crates/zk-por-core/src/circuits/account_circuit.rs

```diff
@@ -0,0 +1,53 @@
+use plonky2::{iop::{target::Target, witness::{PartialWitness, WitnessWrite}}, plonk::circuit_builder::CircuitBuilder};
+
+use crate::{core::account::Account, types::{D, F}};
+
+use super::circuit_utils::assert_non_negative_unsigned;
+
+#[derive(Debug, Clone)]
+/// Targets representing a users account, where their assets and liabilities are split into individual tokens.
+pub struct AccountTargets{
+    pub assets: Vec<Target>,
```

Comment:
i think we use equity, debt, as per professional accounting term. asset = equity - debt

Reply:
Ok will change this